File: ios/Util/EventParameter.m
#import "EventParameter.h"

@implementation EventParameter

NSString * const kKeyIdentifier = @"identifier";
NSString * const kKeyErrorIdentifier = @"errorIdentifier";
NSString * const kKeyDraweOpenCloseSognalState = @"openCloseSignal";
NSString * const kKeyInputDeviceData = @"data";
NSString * const kKeyInterfaceType = @"interfaceType";
NSString * const kKeyConnectionIdentifier = @"connectionIdentifier";
NSString * const kKeyModel = @"model";
NSString * const kKeyEmulation = @"emulation";
NSString * const kKeyReserved = @"reserved";
NSString * const kKeyLanMacAddress = @"macAddress";
NSString * const kKeyLanIPAddress = @"ipAddress";
NSString * const kKeyBluetoothPortName = @"bluetoothPortName";
NSString * const kKeyBluetoothSerialNumber = @"bluetoothSerialNumber";
NSString * const kKeyBluetoothAddress = @"bluetoothAddress";
NSString * const kKeyBluetoothLEAddress = @"bluetoothLEAddress";
NSString * const kKeyUsbPortName = @"usbPortName";
NSString * const kKeyUsbProductSN = @"productSerialNumber";
NSString * const kKeyUsbUsbSN = @"usbSerialNumber";

NSString * const kNamePrinterDelegateReady = @"PrinterReady";
NSString * const kNamePrinterDelegateError = @"PrinterError";
NSString * const kNamePrinterDelegatePaperReady = @"PrinterPaperReady";
NSString * const kNamePrinterDelegatePaperNearEmpty = @"PrinterPaperNearEmpty";
NSString * const kNamePrinterDelegatePaperEmpty = @"PrinterPaperEmpty";
NSString * const kNamePrinterDelegateCoverOpened = @"PrinterCoverOpened";
NSString * const kNamePrinterDelegateCoverClosed = @"PrinterCoverClosed";
NSString * const kNamePrinterDelegateCommunicationError = @"PrinterCommunicationError";
NSString * const kNameDrawerDelegateOpenCloseSignalSwitched = @"DrawerOpenCloseSignalSwitched";
NSString * const kNameDrawerDelegateCommunicationError = @"DrawerCommunicationError";
NSString * const kNameInputDeviceDelegateConnected = @"InputDeviceConnected";
NSString * const kNameInputDeviceDelegateDisconnected = @"InputDeviceDisconnected";
NSString * const kNameInputDeviceDelegateDataReceived = @"InputDeviceDataReceived";
NSString * const kNameInputDeviceDelegateCommunicationError = @"InputDeviceCommunicationError";
NSString * const kNameDisplayDelegateConnected = @"DisplayConnected";
NSString * const kNameDisplayDelegateDisconnected = @"DisplayDisconnected";
NSString * const kNameDisplayDelegateCommunicationError = @"DisplayCommunicationError";
NSString * const kNamePrinterFound = @"PrinterFound";
NSString * const kNameDiscoveryFinished = @"DiscoveryFinished";

@end

File: android/src/main/java/com/stario10module/EventParameter.kt
package com.stario10module

class EventParameter {
    companion object {
        const val KEY_IDENTIFIER = "identifier"
        const val KEY_ERROR_IDENTIFIER = "errorIdentifier"
        const val KEY_DRAWER_OPEN_CLOSE_SIGNAL_STATE = "openCloseSignal"
        const val KEY_INPUT_DEVICE_DATA = "data"
        const val KEY_INTERFACE_TYPE = "interfaceType"
        const val KEY_MODEL = "model"
        const val KEY_EMULATION = "emulation"
        const val KEY_RESERVED = "reserved"
        const val KEY_CONNECTION_IDENTIFIER = "connectionIdentifier"

        const val NAME_PRINTER_DELEGATE_READY = "PrinterReady"
        const val NAME_PRINTER_DELEGATE_ERROR = "PrinterError"
        const val NAME_PRINTER_DELEGATE_PAPER_READY = "PrinterPaperReady"
        const val NAME_PRINTER_DELEGATE_PAPER_NEAR_EMPTY = "PrinterPaperNearEmpty"
        const val NAME_PRINTER_DELEGATE_PAPER_EMPTY = "PrinterPaperEmpty"
        const val NAME_PRINTER_DELEGATE_COVER_OPENED = "PrinterCoverOpened"
        const val NAME_PRINTER_DELEGATE_COVER_CLOSED = "PrinterCoverClosed"
        const val NAME_PRINTER_DELEGATE_COMMUNICATION_ERROR = "PrinterCommunicationError"
        const val NAME_DRAWER_DELEGATE_OPEN_CLOSE_SIGNAL_SWITCHED = "DrawerOpenCloseSignalSwitched"
        const val NAME_DRAWER_DELEGATE_COMMUNICATION_ERROR = "DrawerCommunicationError"
        const val NAME_INPUT_DEVICE_DELEGATE_CONNECTED = "InputDeviceConnected"
        const val NAME_INPUT_DEVICE_DELEGATE_DISCONNECTED = "InputDeviceDisconnected"
        const val NAME_INPUT_DEVICE_DELEGATE_DATA_RECEIVED = "InputDeviceDataReceived"
        const val NAME_INPUT_DEVICE_DELEGATE_COMMUNICATION_ERROR = "InputDeviceCommunicationError"
        const val NAME_DISPLAY_DELEGATE_CONNECTED = "DisplayConnected"
        const val NAME_DISPLAY_DELEGATE_DISCONNECTED = "DisplayDisconnected"
        const val NAME_DISPLAY_DELEGATE_COMMUNICATION_ERROR = "DisplayCommunicationError"
        const val NAME_PRINTER_FOUND = "PrinterFound"
        const val NAME_DISCOVERY_FINISHED = "DiscoveryFinished"
    }
}